.travis.yml

```diff
@@ -2,6 +2,14 @@ language: java
 
 jdk:
   - openjdk8
+  - openjdk9
+  - openjdk10
+  - openjdk11
+  - openjdk-ea
+
+#matrix:
+#  allow_failures:
+#    - jdk: openjdk-ea
```

Comment:
need to check if it is needed (uncomment) or not (remove)

Reply:
We may need this because EA versions of the jdk may be buggy. We shouldn’t fail the build in such cases